libcloud/http.py

```diff
@@ -194,20 +194,22 @@ def __init__(self, host, port, secure=None, **kwargs):
         http_proxy_url_env = os.environ.get(HTTP_PROXY_ENV_VARIABLE_NAME,
                                             https_proxy_url_env)
 
-        # Connection argument rgument has precedence over environment variables
+        # Connection argument has precedence over environment variables
         proxy_url = kwargs.pop('proxy_url', http_proxy_url_env)
 
         self._setup_verify()
         self._setup_ca_cert()
 
         LibcloudBaseConnection.__init__(self)
 
+        self.session.timeout = kwargs.pop('timeout', 60)
```

Comment:
Does this change the default ``requests`` timeout behavior?

If so, we should probably only set that attribute if ``timeout`` is explicitly specified (for backward compatibility reasons and to make sure it doesn't negatively affect long running and streaming connections)?

Reply:
EDIT: Never mind, I see you just moved this statement a couple of lines above, please ignore my comment.